dashboard/app/controllers/level_starter_assets_controller.rb

```diff
@@ -10,7 +10,7 @@ class LevelStarterAssetsController < ApplicationController
 
   # GET /level_starter_assets/:level_name
   def show
-    starter_assets = (@level.starter_assets || []).map do |friendly_name, uuid_name|
+    starter_assets = (@level&.project_template_level&.starter_assets || @level.starter_assets || []).map do |friendly_name, uuid_name|
```

Comment:
could we pull this into a helper method in this file? something like

```ruby
def all_starter_assets
  @level&.project_template_level&.starter_assets || @level.starter_assets
end
```

also, should this be an `||` statement? i'm unfamiliar with template levels, but i would think we would want to combine the template's assets with the individual level's assets

Reply:
In other template levels fields, we overwrite rather than merge. It seemed prudent to follow this pattern to keep consistency. 
Here's the description on the levelbuilder edit form: 
![image](https://user-images.githubusercontent.com/8324574/128948724-8f0213b5-cf48-4e5e-a521-0679cef32732.png)

Reply:
Good call on the helper method. I'll do that in the follow-up PR

Reply:
sounds good -- thanks, jessie!